crates/bin/pindexer/src/dex/dex.sql

```diff
@@ -113,3 +115,21 @@ CREATE TABLE IF NOT EXISTS dex_lp_execution (
   -- The end asset for this execution.
   context_end BYTEA NOT NULL
 );
+
+--- Represents instances where swap executions happened.
+CREATE TABLE IF NOT EXISTS swap
+(
+    height      BIGINT PRIMARY KEY,
+    input       Value,
+    output      Value,
```

Comment:
I think these are redundant if we have the delta amounts

Reply:
removed.